Add workspace symbol configuration

mosswg opened the pull request:
Added the ability to configure the symbols of the workspaces in the .config. The syntax is:
```
WorkspaceSymbol-[Workspace Index]: [Symbol]
```
For example, to change the third workspace to be "3":
```
WorkspaceSymbol-3: 3
```
This indexes from 1, but I can easily change that if you prefer indexing from 0. 

I also added a default workspace symbol option, which is used when no workspace symbol is provided. If no default is provided, then the circle with dot symbol is used. The syntax for changing the default is:
```
DefaultWorkspaceSymbol: [Symbol]
```
The default and a workspace symbol change has been added to the config at data/config.

scorpion-26 commented on src/System.cpp:
```diff
@@ -83,6 +96,18 @@ namespace System
 
+    std::string GetWorkspaceSymbol(int index) {
+        if (index < 0 || index > 9) {
+            return "";
```
This should either ASSERT, or LOG an error, since it is never expected to be in that range and clearly an error from the programmer.

scorpion-26 commented on src/System.h:
```diff
@@ -5,6 +5,8 @@
 
 namespace System
 {
+    std::string GetWorkspaceSymbol(int index);
```
Can you put that to the other Workspace related stuff? (i.e. inside of the Hyprland #ifdef)

scorpion-26 commented on data/config:
```diff
@@ -3,3 +3,5 @@ SuspendCommand: ~/.config/scripts/sys.sh suspend
 LockCommand: ~/.config/scripts/sys.sh lock
 ExitCommand: killall Hyprland
 BatteryFolder: /sys/class/power_supply/BAT1
+WorkspaceSymbol-0: 
```
I think, that should be 1...
Since the loop starts at 1.

scorpion-26 commented:
> This indexes from 1, but I can easily change that if you prefer indexing from 0.
Clicking on a workspace currently adds 1 to the internal index, so that's fine.

mosswg commented:
I've added a log output on index out of bounds including the invalid index in the log. I've also moved the "GetWorkspaceSymbol" function into the Hyprland ifdef and I fixed the mistake in the config. Let me know if there is anything else I should change.

scorpion-26 commented:
LGTM, thanks!
